scripts/travis-setup.sh

```diff
@@ -23,7 +23,7 @@ _setup_deps() {
   export NVIM_DEPS_PREFIX="${1}/usr"
   echo "\$NVIM_DEPS_PREFIX: \"${NVIM_DEPS_PREFIX}\""
 
-  eval "$(${NVIM_DEPS_PREFIX}/bin/luarocks path)"
+  "$(${NVIM_DEPS_PREFIX}/bin/luarocks path)"
```

Comment:
This one is also different. Before it was setting LUA_PATH and LUA_CPATH. Now you are trying to execute `export LUA_PATH=…\nexport LUA_CPATH=…` *as a single command* and, of course, this is not going to work.